src/osg_display/osg_display.py

```diff
@@ -222,18 +223,19 @@ def main():
     log.info("OSG Display done!")
     log.debug("Time log - Total Time: %s", (time.time() - watchB))
 
-main_unwrapped = main
-
 def main():
-
-    try:
-        main_unwrapped()
-    except SystemExit:
-        raise
-    except (Exception, KeyboardInterrupt), e:
-        log.error(str(e))
-        log.exception(e)
-        raise
```

Comment:
Seems you are dropping the exception logging - is this intentional?

Reply:
Dropping the catch-all was intentional. It seemed only needed to get the exceptions into the logfile.